public/locales/en/common.json

```diff
@@ -7,6 +7,50 @@
   "developer": "Developer",
   "about": "About",
   "language": "中文",
-  "lets-talk": "Contact With Us",
-  "login": "Login"
+  "lets-talk": "Appointment",
+  "login": "Login",
+  "footer-menu-1-title": "Products",
+  "footer-menu-1-1-title": "Large Scale Customer Acquisition",
+  "footer-menu-1-2-title": "Marketing Automation",
+  "footer-menu-1-3-title": "Customer Conversation Panel",
+  "footer-menu-1-4-title": "Data Panel",
+  "footer-menu-1-5-title": "Management Center",
+  "footer-menu-1-6-title": "Security Compliance",
+  "footer-menu-1-7-title": "Cloud Native",
+  "footer-menu-1-8-title": "RPA",
+  "footer-menu-1-9-title": "Privatization deployment",
+  "footer-menu-2-title": "Sulotions",
+  "footer-menu-2-1-title": "Private Domain",
+  "footer-menu-2-2-title": "Customer Service",
+  "footer-menu-2-3-title": "Scalable",
+  "footer-menu-2-4-title": "Concersion",
```

Comment:
We can change `Concersion` to one of the following options:

- Engagement
- Conversational Enagagement
- Customer Engagement